cmd/podman/shared/intermediate_novarlink.go

```diff
@@ -0,0 +1,70 @@
+// +build !varlink
+// +build !remoteclient
```

Comment:
I am not sure this works.  I think these have to be on the same line.

Reply:
no this is correct

Reply:
I think they can be on the same line, but there is some sort of boolean logic that needs to be applied. @baude What does separate lines do with build tags - logical-AND or logical-OR?